tests/testthat/test-removeat.R

```diff
@@ -198,3 +198,47 @@ test_that("RemoveAt: another vector and a list",
                         "NET Sugarred", "NET Sugarless", "NET"),
                 "Age in years"))
 })
+
+# Check flattened tables are filtered correctly
+
+ftbls <- readRDS("ftbls.rds")
```

Comment:
If the structure of ftables changes in a later version of R, your tests won't pick that up.

Reply:
Updated.